lightkurve/interact.py

```diff
@@ -340,6 +350,7 @@ def create_interact_ui(doc):
                                 width=490)
         r_button = Button(label=">", button_type="default", width=30)
         l_button = Button(label="<", button_type="default", width=30)
+        export_button = Button(label="Export Mask", button_type="success", width=120)
```

Comment:
Should this button be named `Save Lightcurve`?  (Given that this button is really creating a new light curve file with aperture mask extension, which I think is the right thing to do!)

Reply:
Ideally, yes.  As it presently stands, lightkurve offers no convenient way to read in the product of a `to_fits()` operation (see #289), at least not while preserving metadata.  The word `mask` was chosen over `lightcurve` to encourage a pattern of mask reuse, rather than lightcurve reuse, for now.  I had considered zeroing out the time and flux hdu extension as a safeguard, too, but decided a nudge was adequate.

Reply:
We have a unit test which tests that the output of `LightCurve.to_fits()` can be read in by `KeplerLightCurveFile`.  It's annoying that `.PDCSAP_FLUX` won't necessarily work for such object (not written by default), but otherwise we have a well-functioning way to read in data written using `to_fits()`, I think!  I'd go ahead and make this a full `Save Lightcurve` feature, and work on improving the generic `LightCurveFile` class in a separate PR?